modules/google-analytics/wp-google-analytics.php

```diff
@@ -58,6 +59,7 @@ private function __construct() {
 			$analytics = new Jetpack_Google_Analytics_Legacy();
 		}
 
+		new Jetpack_Google_AMP_Analytics();
```

Comment:
Since this is about WooCommerce tracking, should we place this inside the `class_exists( 'WooCommerce' ) && Jetpack_Google_Analytics_Options::enhanced_ecommerce_tracking_is_enabled()` check above?